db/migrate/20210421064041_add_action_to_report.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddActionToReport < ActiveRecord::Migration[5.2]
+  def change
+    add_column :reports, :action, :string
+
+    Report.update_all(action: Report::STATUS_NO_ACTION)
```

Comment:
Rails/SkipsModelValidations: Avoid using `update_all` because it skips validations.